docs/docs/sourcing-from-seams-cms.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Sourcing from Seams-CMS
+---
+
+## Overview
+
+[Seams-CMS](https://seams-cms.com) is a headless CMS system that allows you to easily manage content and data with either simple or complex structures and supports localization.
+
+Creating a Seams-CMS account is free for hobby or small sites. For this tutorial, we will be using the demo workspace that is automatically generated when you create a new account. Make sure you add an API key to this workspace in the API key settings. We will be needing your workspace ID and the (read-only) API key shortly.
+
+## Setup
+
+Create a new Gatsby site with the [default starter](https://github.com/gatsbyjs/gatsby-starter-default):
+
+Run this in your terminal:
+
+```shell
+gatsby new seamscms-site
+cd seamscms-site
+```
+
+### Install the source plugin
+
+Next, we will be installing the Seams-CMS Gatsby source plugin:
+
+```shell
+npm install --save  seams-cms-gatsby-source
+```
+
```

Comment:
What do you think about rolling "Install the source plugin" into the previous section? Something like:

## Setup

Create a new Gatsby site with the [default starter](https://github.com/gatsbyjs/gatsby-starter-default) and install the Seams-CMS source plugin.

Run this in your terminal:

```shell
gatsby new seamscms-site
cd seamscms-site
npm install --save seams-cms-gatsby-source